test/kata2.booleanToWord.test.js

```diff
@@ -1,5 +1,13 @@
 const { booleanToWord } = require("../src");
 
 describe("booleanToWord", () => {
-  // how do we create specs again???
+  it("should return 'Yes' if passed true", () => {
+    expect(booleanToWord(2 * 2 === 4)).toEqual("Yes");
+    expect(booleanToWord(1 == "1")).toEqual("Yes");
+  });
+
+  it("should return 'No' if passed false", () => {
+    expect(booleanToWord(2 * 2 === 5)).toEqual("No");
+    expect(booleanToWord(1 === "1")).toEqual("No");
+  });
```

Comment:
These tests will need refactoring. Basically, the idea is to pass either true or false, or a truthy/falsy value, not necessarily a comparison.